### PR TITLE
chore(docs): Added the autobuild option to instructions

### PR DESCRIPTION
**Pull request by bethcg**
Updated docs with auto build instruction.

### CODE REVIEW

**Review by rokroskar — DISMISSED** [review]
Thanks @bethcg!

**Comment by rokroskar** [comment]
Please add a description to the PR - otherwise one of the CI jobs fails ;)

File: docs/how-to-guides/contributing/documentation.rst [hunk 1]
```diff
@@ -44,4 +53,4 @@ Using Nix
 
 We provide a ``nix`` configuration that provides all the required libraries for developing the
 renku helm charts and the documentation. To use, simply run ``nix-shell`` and your shell will
-be configured with everything you need.
+be configured with everything you need.
```
Comment (aledegano) [inline docs/how-to-guides/contributing/documentation.rst:56]: Nitpicking : could you please preserve the newline at the end of the file? 😇

Comment (bethcg) [reply docs/how-to-guides/contributing/documentation.rst]: Done!

**Review by aledegano — APPROVED** [review]
Thanks for improving the docs!

**Pull request merged**